src/Transactions/certificates.ts

```diff
@@ -123,11 +123,24 @@ join pool_hash
   on pool.hash_id = pool_hash.id
 join stake_address as addr
   on addr.hash_raw = pool.reward_addr
-left join pool_meta_data as pool_meta
+left join pool_metadata_ref as pool_meta
   on pool_meta.id = pool.meta_id
+group by pool.registered_tx_id
```

Comment:
I think the grouping is necessary because otherwise we get a SQL error: `column "{column_name}" must appear in the GROUP BY clause or be used in an aggregate function`.

I'm not aware of some way to circumvent this to allow grouping only by the primary key in these cases where we have several columns being selected together with an aggregate function

Reply:
make sense, thanks for explaining